docs/faq.md

```diff
@@ -96,7 +96,7 @@ https://github.com/rancher-sandbox/rancher-desktop/issues
 <!-- RD #1262 -->
 #### **Q: I can no longer run `docker compose` after installing Rancher Desktop and uninstalling Docker Desktop, what happened?**
 
-**A:** The `docker compose` subcommand is bundled as part of a Docker Desktop installation and is removed when uninstalled. Future versions of Rancher Desktop will include and automatically `docker compose` for you, but until then you can install it using this procedure:
+**A:** The `docker-compose` utility is bundled as part of a Docker Desktop installation and is removed when uninstalled. Rancher Desktop version 1.1.0 and above comes bundled with `docker-compose` for you. If you don't see `docker-compose` available then please file a bug on [Github](https://github.com/rancher-sandbox/rancher-desktop/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml). Also, the steps to manually install `docker-compose` is provided below for reference:
```

Comment:
That should be rewritten, as 1.3.0 installs into `~/.docker/cli-plugins` for you. I'm not sure if the manual install instructions are still accurate; they may need to be updated (or removed?).